Competition2018/src/test/java/competition/subsystems/climberdeploy/Command/Tests/StopClimberArmCommandTest.java

```diff
@@ -0,0 +1,41 @@
+package competition.subsystems.climberdeploy.Command.Tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.subsystems.climberdeploy.ClimberdeploySubsystem;
+import competition.subsystems.climberdeploy.commands.StopClimberArmCommand;
+import xbot.common.controls.actuators.mock_adapters.MockCANTalon;
+import xbot.common.injection.BaseWPITest;
+
+public class StopClimberArmCommandTest extends BaseWPITest{
+	
+	ClimberdeploySubsystem deploy;
+	StopClimberArmCommand command;
+	
+	@Override
+	public void setUp() {
+		super.setUp();
+		
+		deploy = injector.getInstance(ClimberdeploySubsystem.class);
+		command = injector.getInstance(StopClimberArmCommand.class);
+		
+		deploy.temporaryHack();
+	}
+	
+	@Test
+	public void simpleTest() {
+		command.initialize();
+		command.execute();
+	}
+	
+	@Test
+	public void checkStopClimberArm() {
+		command.initialize();
+		command.execute();
+		
+		assertEquals(0.0, deploy.motor.getMotorOutputPercent(), 0.001);
```

Comment:
There's no way to know if your command works, since the speed was probably already 0 beforehand. You need your test to make the system move first, then run your command to stop it.

Reply:
I've poked around with trying to import motors and prop men but I still can't figure out how to change the state of the system, as opposed to just looking at it, in a Test Command.